src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -287,6 +288,76 @@
     </ItemGroup>
   </Target>
 
+  <!--
+    Condition should be removed after source-build picks up version of Arcade that has
+    source-build infra changes for removal od inner-clone in VMR build. Until then some
+    repos will produce intermediate packages instead of repo manifest file.
+
+    https://github.com/dotnet/source-build/issues/3930
+  -->
+  <Target Name="CopyRepoArtifacts"
+          AfterTargets="Package"
+          Condition="Exists($(RepoManifestFile))"
+          Inputs="$(RepoManifestFile)"
+          Outputs="$(BaseIntermediateOutputPath)ArtifactsCopy.complete">
+    <XmlPeek XmlInputPath="$(RepoManifestFile)"
+             Query="Build/Artifact/@Path">
+        <Output TaskParameter="Result" ItemName="RepoManifestArtifact" />
+    </XmlPeek>
+
+    <ItemGroup>
+      <_Asset Include="@(RepoManifestArtifact)" Condition="'%(Extension)' != '.nupkg'"/>
+      <_Package Include="@(RepoManifestArtifact)" Condition="'%(Extension)' == '.nupkg'"/>
+      <_NonShippingPackage Include="@(_Package)" Condition="$([System.String]::Copy('%(Identity)').Contains('$([System.IO.Path]::DirectorySeparatorChar)NonShipping$([System.IO.Path]::DirectorySeparatorChar)'))"/>
```

Comment:
Use less ambiguous item names here as well: RepoManifestAsset, RepoManifestPackage and RepoManifestNonShippingPackage.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18153/commits/5921162d7049612306fcd3785de068af873946af